site/src/docs/components/header/index.mdx

```diff
@@ -22,56 +22,232 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
+- The component supports up to 2 levels of nested links. 
+- On smaller screens (768px and down) the Header.NavigationMenu is hidden and the contents are moved inside the [the Header.ActionBar component's](#with-action-bar) menu.
 
-Coming soon!
+<PlaygroundPreview>
+  <Header>
+    <Header.ActionBar
+      title="Helsingin kaupunki"
+      titleHref="https://hel.fi"
+      logoAriaLabel="Service logo"
+      logoHref="https://hel.fi"
+      menuButtonAriaLabel="Menu"
+    >
+      <Header.NavigationLanguageSelector languages={[
+        { label: 'Suomi', value: 'fi' },
+        { label: 'Svenska', value: 'sv' },
+        { label: 'English', value: 'en' },
+      ]} />
+      <Header.ActionBarItem fullWidth label="Haku" icon={IconSearch} id="action-bar-search">
+        <Header.NavigationSearch label="Hae palvelusta" onChange={() => {}} onSubmit={() => {}} />
+      </Header.ActionBarItem>
+    </Header.ActionBar>
+    <Header.NavigationMenu>
+      <Header.NavigationLink
+        href="#"
+        label="Link 1"
+        onClick={(event) => event.preventDefault()}
+        active
+        dropdownLinks={[
+          <Header.NavigationLink
+            href="#"
+            label="Nested link 1"
+            active
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Deep link 1" />,
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+              <Header.NavigationLink href="#" label="Deep link 3" />,
+            ]}
+          />,
+          <Header.NavigationLink
+            href="#"
+            label="Nested link 2"
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Deep link 1" />,
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+            ]}
+          />,
+        ]}
+      />
+      <Header.NavigationLink
+        href="#"
+        label="Link 2"
+        dropdownLinks={[
+          <Header.NavigationLink
+            href="#"
+            label="Nested link 1"
+            active
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Deep link 1" />,
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+            ]}
+          />,
+          <Header.NavigationLink
+            href="#"
+            label="Nested link 2"
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Deep link 1" />,
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+            ]}
+          />,
+        ]}
+      />
+      <Header.NavigationLink href="#" label="Link 3" />
+    </Header.NavigationMenu>
+  </Header>
+</PlaygroundPreview>
 
-#### With navigation bar
+#### With universal bar
 
```

Comment:
Universal bar is aimed for frequenlty visited and useful links, e.g. feedback and news.